test/profiler/test_profiler_tree.py

```diff
@@ -690,6 +690,7 @@ def forward(self, x: torch.Tensor) -> torch.Tensor:
                   ...""",
         )
 
+    @skipIfTorchDynamo("segfaults in 3.13+")
```

Comment:
This happened with a similar `torch_dispatch` test below: https://github.com/pytorch/pytorch/blob/10c4c45957546be5e1a86590834baaef68bba334/test/profiler/test_profiler_tree.py#L734-L739